porch/controllers/workloadidentitybinding/pkg/controllers/workloadidentitybinding/controller.go

```diff
@@ -45,46 +50,57 @@ type WorkloadIdentityBindingReconciler struct {
 //+kubebuilder:rbac:groups=config.porch.kpt.dev,resources=workloadidentitybindings/finalizers,verbs=update
 //+kubebuilder:rbac:groups=,resources=namespaces,verbs=get;list;watch
 //+kubebuilder:rbac:groups=iam.cnrm.cloud.google.com,resources=iampolicymembers,verbs=get;list;watch;create;update;patch;delete
+//+kubebuilder:rbac:groups=,resources=serviceaccounts,verbs=get;list;watch;create;update;patch;delete
```

Comment:
Unclear what permissions we need, but probably just get;list;watch and either update or patch.

Reply:
Fixed.